doc/release-notes-30592.md

```diff
@@ -0,0 +1,7 @@
+Full Replace-By-Fee
+===================
+
+Starting with v28.0, the `mempoolfullrbf` startup option was set to
+default to `1`. With widespread adoption of this policy, users no longer
+benefit from disabling it, so the option has been removed, making full
+replace-by-fee the standard behavior. (#30592)
```

Comment:
This is a bit confusing, because defaulting to `mempoolfullrbf=1` already made full replace-by-fee the standard behavior, but this reads as if removing the option makes it the standard behavior.
Perhaps something like this:

```suggestion
After the policy was broadly adopted by miners, starting with v28.0, the `mempoolfullrbf` startup option was set to 
default to `1`. With widespread adoption of full replace-by-fee, it becomes the standard behavior on the network. Users no longer benefit from disabling it, so the option has been removed.
```

Reply:
will take language if I touch PR again